django_netjsongraph/base/node.py

```diff
@@ -8,12 +8,17 @@
 from django.utils.timezone import now
 from jsonfield import JSONField
 from rest_framework.utils.encoders import JSONEncoder
+from sortedm2m.fields import SortedManyToManyField
 
 from .. import settings
 from ..utils import print_info
 from .base import TimeStampedEditableModel
 
 
+class Address(models.Model):
+    address = models.GenericIPAddressField(primary_key=True)
```

Comment:
some graphs use layer 2 addresses or non-ip addresses as well, so we cannot use `GenericIPAddressField`, this needs to be a charfield (maxlength can be the maximum legth of an ipv6 address).

The primary key also may need to be implemented differently, because there may be the same address in 2 different topologies, ensure this is doable (write a test for it).